src/components/DocNode/DocNode.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { FC, ReactNode, ReactElement } from 'react';
+import { GuidanceState, DocumentationResource, State } from 'pathways-model';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import classes from './DocNode.module.scss';
+import indexClasses from 'styles/index.module.scss';
+
+interface DocNodeProps {
+  pathwayState: GuidanceState | State;
+  isActionable: boolean;
+  documentation: DocumentationResource | undefined;
+  isGuidance: boolean;
+}
+
+const DocNode: FC<DocNodeProps> = ({ pathwayState, isActionable, documentation, isGuidance }) => {
+  let fhirFields: ReactElement[] = [];
+  if (documentation && documentation.resource) {
+    fhirFields = parseResource(documentation.resource);
+  }
+  const guidance = isGuidance && renderRecGuidance(pathwayState as GuidanceState);
+
+  return (
+    <div className={indexClasses.docNode}>
+      <table className={classes.infoTable}>
+        <tbody>
+          {guidance}
+          {fhirFields}
+        </tbody>
+      </table>
+      {isActionable && (
+        <form className={classes.commentsForm}>
+          <label>Comments:</label>
+          <button className={indexClasses.button} onClick={(e): void => e.preventDefault()}>
+            Use Default Text
+          </button>
+          <textarea className={classes.comments}></textarea>
+          <button
+            className={`${classes.accept} ${indexClasses.button}`}
+            onClick={(e): void => e.preventDefault()}
+          >
+            Accept
+          </button>
+          <button
+            className={`${classes.decline} ${indexClasses.button}`}
+            onClick={(e): void => e.preventDefault()}
+          >
+            Decline
+          </button>
+        </form>
+      )}
+    </div>
+  );
+};
+
+type DocNodeFieldProps = {
+  title: string;
+  description: ReactNode;
+};
+
+const DocNodeField: FC<DocNodeFieldProps> = ({ title, description }) => {
+  return (
+    <tr>
+      <td className={classes.descTitle}>{title}</td>
+      <td className={classes.desc}>{description}</td>
+    </tr>
+  );
+};
+
+function renderRecGuidance(pathwayState: GuidanceState): ReactElement[] {
```

Comment:
I would make this a React FC (like DocNodeField).  That way we can still call this with JSX syntax.

See [React Conditional Rendering](https://reactjs.org/docs/conditional-rendering.html)

i.e. `renderRecGuidance` vs `<RecGuidance/>`